trio/_core/_tests/test_run.py

```diff
@@ -1598,7 +1594,7 @@ async def test_asyncio_function_inside_nursery_does_not_explode():
     # Regression test for https://github.com/python-trio/trio/issues/552
     with pytest.raises(TypeError) as excinfo:
         async with _core.open_nursery() as nursery:
-            import asyncio
+            # import asyncio
```

Comment:
I'm not sure if this `import` is to ensure `asyncio` is available or something

Reply:
I think this is just missed in PR review. cc @oremanj as author of https://github.com/python-trio/trio/pull/1178 even though that was.... 4 years ago wow.